test/test.wpcom.site.follow.js

```diff
@@ -7,7 +7,7 @@ var assert = require( 'assert' );
 /**
  * site.follow
  */
-describe( 'wpcom.site.follow', function() {
+describe.skip( 'wpcom.site.follow', function() {
```

Comment:
Was this intended to be committed? Are these tests unfixable?

Several describe blocks are skipped by this PR, that concerns me.

Reply:
Hi @sirreal - this was an issue with the tests rather than this PR, they relied on production state & data and they were previously broken.

Reply:
Thanks @nylen. I hadn't seen there was already an issue open to fix them: https://github.com/Automattic/wpcom.js/issues/218